tendrl/node_agent/node_sync/__init__.py

```diff
@@ -59,8 +59,9 @@ def _run(self):
 
                 # updating node context with latest tags
                 LOG.info("node_sync, updating node context data with tags")
-                tags += NS.node_context.tags
-                NS.tendrl.objects.NodeContext(tags=tags).save()
+                tags += NS.node_context.tags.split(",")
```

Comment:
Please create in node agent conf file

tags:
- tendrl/node

Etcd will contain tags stored as json array, hence no need to specially process them